packages/api/cms-api/package.json

```diff
@@ -66,6 +66,7 @@
         "reflect-metadata": "^0.1.0",
         "rimraf": "^3.0.0",
         "slugify": "^1.6.4",
+        "svg-parser": "^2.0.4",
```

Comment:
This package was last updated four years ago... shouldn't we use something more up-to-date?

<img width="380" alt="image" src="https://github.com/vivid-planet/comet/assets/48853629/3bc89de1-4d37-41a9-85b5-ce7da709b255">

Can't we use [fast-xml-parser](https://www.npmjs.com/package/fast-xml-parser), as SVGs are XML based?

Reply:
Yes I know. I still chose it over other svg parsers because it's zero dependency. So if it works, it works and the long time without publish isn't necessarily a security risk.

But I'll check out the fast-xml-parser

Reply:
I implemented fast-xml-parser now